modules/admin_manual/pages/installation/manual_installation/script_guided_install.adoc

```diff
@@ -101,11 +124,11 @@ the script extracts the files with tar and automatically extracts them to the ta
 .In case of an upgrade
 the old instance path is backed up by _renaming and adding a time stamp_. A new target folder with the old name is created. This ensures that in case of extracting and preparation issues, you can easily go back to the previous version. After a successful upgrade, you must manually remove the backup folder.
 
-TIP: If you do not install or upgrade, the script sets ownership and permissions only.
+TIP: If you do not install or upgrade, the script sets ownership and permissions on linked directories only.
 
 NOTE: In case of upgrading, if you have customized the `.htaccess` or `.user.ini` file in the ownCloud webroot, you have to manually restore any changes made after the script has finished from the backup folder. Take care to only restore the changes made but not the complete file as this file will be recreated on upgrades by the tar source and may contain different settings provided by ownCloud. Customizing `.htaccess` can be necessary when you e.g. xref:configuration/integration/ms-teams.adoc[Integrate ownCloud into Microsoft Teams].
 
-NOTE: In case of upgrading, if you have used links for the apps-external and data directory, the script will relink these directories. If you have used normal directories, you manually have to copy any content of these directories back after the script has run. Note you have take care not to overwrite new app versions with older ones.
+NOTE: In case of upgrading, if you have used links for the apps-external/ and data/ directory, the script will relink these directories. If you have used standard directories, you manually have to migrate any content of these directories back after the script has run. Check that ownership and permissions are correct after migration and in doubt, let the scrips set the correct ones when using linked directories or ste them manually - which can take a while in any case depending on the size of the data/ directory. Note you have take care not to overwrite new app versions with older ones.
```

Comment:
```suggestion
NOTE: In case of upgrading, if you have created links to the apps-external/ and data/ directory, the script will re-link these directories. If you used standard directories, you have to manually migrate any content of these directories back after the script has run. Check that ownership and permissions are correct after migration and in doubt, let the scrips set the correct ones when using linked directories or set them manually - which can take a while depending on the size of the data/ directory. Take care not to overwrite new app versions with older ones.
```